src/main/java/seedu/address/model/sampledata/SampleLocalCourses.java

```diff
@@ -0,0 +1,1298 @@
+            + " aims to teach students concepts, methods and tools to demonstrate the return on investment (ROI) of "
+            + "marketing activities and to leverage on data and marketing analytics to make better and more informed "
+            + "marketing decisions. Course topics covered include marketing performance management, marketing "
+            + "metrics, data management, market response and diffusion models, market and customer segmentation "
+            + "models, analytic marketing and value driven segmentation, digital media marketing analytics, etc. "
+            + "Students will have access to"),
+        BT4212("BT4212", "Search Engine Optimization and Analytics",
+            "This course teaches the concepts, techniques and methods to analyse and improve the visibility of a "
+            + "website or a web page in search engines via the “natural” or un-paid (“organic” or “algorithmic”) "
+            + "search results. Students will be taught concepts and knowledge in terms of how search engines work, "
+            + "what people search for, what are the actual search terms or keywords typed into search engines, which "
+            + "search engines are preferred by their targeted audience, and how to optimize a website in terms of "
+            + "editing its content, structure and links, and associated coding to both increase its relevance to "
+            + "specific keywords and to remove barriers to the indexing activities of search engines. Importantly, "
+            + "the course will emphasize the relationship of search engine optimization to digital marketing in terms"
+            + " of building high quality web pages to engage and persuade, setting up analytics programs to enable "
+            + "sites to measure results, and improving a site's conversion rate."),
+        BT4221("BT4221", "Big Data Techniques and Technologies",
+            "This course teaches students concepts, techniques and technologies in handling and analyzing big data "
+            + "sets, i.e., data sets whose size is beyond the ability of commonly used software tools to capture, "
+            + "manage, and process the data within a tolerable elapsed time. Common sources and domains of big data "
+            + "include ubiquitous information-sensing mobile devices, web and software logs, financial trading "
+            + "transactions, large-scale e-commerce, RFID and wireless sensor networks, etc. Conceptual big data "
+            + "topics covered include big data instructure, analytics scalability and processes, etc. Technical "
+            + "frameworks for big data handling and analytics taught include Hadoop, HBase, Cassandra, MapReduce, "
+            + "Dynamo, R, in-database analytics,"),
+        BT4222("BT4222", "Mining Web Data for Business Insights",
+            "The World Wide Web overwhelms us with immense amounts of widely distributed, interconnected, rich, and "
```

Comment:
Line wrap indentation might be wrong, not too sure again since this likely passed the checkstyle.